src/megacmdshell/megacmdshellcommunications.h

```diff
@@ -40,15 +41,14 @@
 
 #if defined(_WIN32) && !defined(WINDOWS_PHONE) && !defined(USE_CPPTHREAD)
 #include "mega/thread/win32thread.h"
-class MegaMutex : public ::mega::Win32Mutex {};
+class std::mutex : public ::mega::Win32Mutex {};
```

Comment:
I think we can just omit this definition now, as the std library will provide it? (and same below for the cppthread case)    thanks!